extended-statistics.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Introduction to Extended Statistics
+summary: Learn how to use extended statistics to guide the optimizer.
+---
+
+# Introduction to Extended Statistics
+
+The statistics mentioned in the [Introduction to Statistics](/statistics.md) section, including histograms and Count-Min Sketch, are regular statistics. This information is collected each time statistics are collected manually or automatically. Another class of statistics, as opposed to common statistics, is extended statistics, which are only helpful for optimizer estimation in a specific scenario.
+
+Since they are only helpful in specific scenarios, extended statistics are not collected during the default manual or automatic `ANALYZE` to avoid the overhead of managing statistics. If you want to collect extended statistics, you need to "register" them with SQL commands first. Then TiDB will collect these registered extended statistics in addition to the regular statistics the next time you manually or automatically `ANALYZE`.
+
+# The registration of the Extended Statistics
+
+If you want to register the extended statistics, you can use the SQL `ALTER TABLE ADD STATS_EXTENDED`. The grammar is shown below:
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE table_name ADD STATS_EXTENDED IF NOT EXISTS stats_name stats_type(column_name, column_name...);
+```
+
+This statement indicates that you want to collect the specified type of extended statistics on the specified columns of the table and name it.
+
+- `table_name` is the table that you want to collect the extended statistics.
+- `stats_name` is the name of the extended statistics. It should be unique for each table.
+- `stats_type` is the type of the extended statistics. Now it only has one possible value `correlation`.
+- `column_name` specifies the column group. It can be multiple columns. For `correlation` type, there should be and only be two columns.
+
+The extended statistics will be collected if the `mysql.stats_extended` has the corresponding record when we run the `ANALYZE` command. And the `status` column will be set to `1`, and the `version` column will be set to the new timestamp.
+
+## The type of the Extended Statistics
+
+### Correlation
+
+This is the only supported type of extended statistics. The registration SQL is like the following:
```

Comment:
I don't think it's necessary to emphasize this is the only option.  Just tell what it is ... 
In the future version, we will have more options included, so you will not have to review the entire page to fix the description.